src/components/datagrid/DataGrid.js

```diff
@@ -60,9 +62,9 @@ export default class DataGridComponent extends NestedArrayComponent {
   get defaultSchema() {
     return DataGridComponent.schema();
   }
-  // Checks if should init the first row (only when no defaultValue and just one row set)
+
   get initRows() {
-    return this.builderMode || !_.isEqual(this.defaultValue, this.emptyValue) || !this.component.noFirstRow;
```

Comment:
This breaks reverse compatibility with forms that already have the old configuration set. Can we change this logic to also check for ```noFirstRow```?

Reply:
Done